src/hooks/competitorData/microsoft_clarity.tsx

```diff
@@ -0,0 +1,82 @@
+export const microsoft_clarity = {
+    name: 'Microsoft Clarity',
+    key: 'microsoft_clarity',
+    assets: {
+        icon: '/images/competitors/microsoft-clarity.svg',
+        comparisonArticle: 'blog/best-microsoft-clarity-alternatives',
+    },
+    products: {
+        session_replay: {
+            available: true,
+            pricing: {
+                free_tier: true, // Completely free forever
+            },
+            features: {
+                canvas_recording: false, // Cannot render canvas elements
+                chat_with_recordings: false, // No AI chat feature
+                clickmaps: true, // Yes
+                conditional_recording: false, // No built-in conditional recording
+                crash_reports: false, // Not available
+                event_timeline: true, // Yes
+                export_to_json: false, // No JSON export for recordings
+                export_to_video: false, // No video export
+                filter_by_user_or_event: true, // Yes, extensive filters
+                identity_detection: true, // Yes, user identification
+                iframe_recording: false, // Cannot record third-party iframes
+                minimum_duration: false, // No minimum duration setting
+                movement_maps: false, // No movement tracking
+                notes_on_replays: true, // Yes, can add notes
+                playlists: false, // No playlists, but has favorites
+                privacy_masking: true, // Yes, automatic masking
+                retention_policy: true, // 30 days standard, 13 months for favorites
```

Comment:
Should this be a string with what the retention policy actually is? Every tool has retention of some sort...